README.md

```diff
@@ -51,6 +53,12 @@ There are `make` commands that alias some of this functionality:
  * `publish` -- Outputs the specification as a **single file** into the `build/` directory
  * `serve` -- Serves a preview of the specification in human-readable format
 
+### Running locally
+To run the swagger api into http://0.0.0.0:5000, the you need to execute:
+ 1. `make install`
+ 2. `make build` created the build into `/build` directory
+ 3. `make serve` from `/build` directory
```

Comment:
Not sure I follow - with the instructions I looked through there is no makefile in `/build` so I run into

```
make: *** No rule to make target `serve'.  Stop.
```